test/extensions/filters/http/ext_proc/client_test.cc

```diff
@@ -79,14 +79,14 @@ class ExtProcStreamTest : public testing::Test, public ExternalProcessorCallback
 };
 
 TEST_F(ExtProcStreamTest, OpenCloseStream) {
-  auto stream = client_->start(*this, config_with_hash_key_, stream_info_);
+  auto stream = client_->start(*this, config_with_hash_key_, Http::AsyncClient::StreamOptions());
```

Comment:
I think this update minor changed the original test. In the original test, the stream info in the ParentContext is non-null. But now, it's null.

Reply:
Nice catch! Thank you.